source/renderer/app/components/news/AlertsOverlay.js

```diff
@@ -58,15 +59,21 @@ export default class AlertsOverlay extends Component<Props, State> {
     this.props.onMarkNewsAsRead(alerts[0].date);
   };
 
+  onProceedNewsAction(event: SyntheticMouseEvent<HTMLElement>) {
```

Comment:
It's always good to create fat-arrow fns instead of class methods, as you don't need to use `.bind` bellow, which causes unnecessary renderings. Unless I missed something and it had to be done this way :)

Reply:
@daniloprates I need `.bind` to pass event because store Fn can accept that event and pass to openExternalLink which will preventDefault() but I agree that I need to use arrow function instead Vanilla fn.

Reply:
👍